bin/sozo/src/commands/test.rs

```diff
@@ -67,9 +28,9 @@ pub struct TestArgs {
     /// Should we run only the ignored tests.
     #[arg(long, default_value_t = false)]
     ignored: bool,
-    /// Should we run the profiler and with what mode.
-    #[arg(long, default_value = "none")]
-    profiler_mode: ProfilerMode,
+    /// Should we run the profiler.
+    #[arg(long, default_value_t = false)]
```

Comment:
No more profiler mode?

Reply:
I don't know, I only found this option: https://foundry-rs.github.io/starknet-foundry/appendix/snforge/test.html#--build-profile